ui/webui/resources/polymer_overriding.js

```diff
@@ -49,7 +49,7 @@ function addBraveProperties(moduleName, component) {
 const allBraveTemplateModificationsMap = {}
 
 function addBraveTemplateModifications(moduleName, component, modifyFn) {
-  const template = component.template || component._template
+  const template = component._template || component.template
```

Comment:
Just curious on the logic change here (as you have a lot more context!). Why would `_template` be preferred?

Reply:
`_template` is where polymer keeps the memoized copy of the template. If it exists, it's already read from `template` and won't read from it again.